joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/dataflow/DataflowTest.scala

```diff
@@ -695,4 +695,16 @@ class DataflowTest extends DataFlowCodeToCpgSuite {
       sink.reachableBy(identifierSource).size shouldBe 1
     }
   }
+  "Field access on TemplatedDom directly" should {
```

Comment:
Could you add a space between the tests, please?

Reply:
done